pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/AdminApiSchemaTest.java

```diff
@@ -247,6 +249,25 @@ public void createKeyValueSchema(ApiVersion version) throws Exception {
         assertEquals(keyValueSchema.getSchemaInfo(), schemaInfo);
     }
 
+
+    @Test(dataProvider = "version")
+    public void testInvalidSchemaDataException(ApiVersion version) {
+        String namespace = format("%s%s%s", "schematest", (ApiVersion.V1.equals(version) ? "/" + cluster + "/" : "/"),
+                "test");
+        String topicName = "persistent://"+ namespace + "/test-invalid-schema-data-exception";
+        SchemaInfo schemaInfo = SchemaInfo.builder()
+                .schema(new byte[0])
+                .type(SchemaType.AVRO)
+                .name("test")
+                .build();
+        try {
+            admin.schemas().createSchema(topicName, schemaInfo);
```

Comment:
Actually we should add the `fail()` call here.

Reply:
Oh, the PR gets merged. I can fix it in other PR.